core/internal/features_test.go

```diff
@@ -873,3 +873,36 @@ func TestIntegration_AuthToken(t *testing.T) {
 	defer cleanup()
 	cltest.AssertServerResponse(t, resp, http.StatusOK)
 }
+
+func TestIntegration_FluxMonitor_Deviation(t *testing.T) {
+	app, cleanup := cltest.NewApplication(t)
+	defer cleanup()
+
+	eth := app.MockCallerSubscriberClient(cltest.Strict)
+	eth.Register("eth_chainId", app.Store.Config.ChainID())
+	app.StartAndConnect()
+
+	// 1. FM gets initial price on chain.
+	eth.Context("Flux Monitor initializes price", func(mock *cltest.EthMock) {
+		mock.Register("eth_call", "10000") // 10,000 cents
+	})
+
+	// 2. FM checks price adapter for deviation, and gets a price with enough deviation.
+	priceResponse := `{"data":{"result": 102}}`
+	mockServer, assertCalled := cltest.NewHTTPMockServer(t, http.StatusOK, "POST", priceResponse)
+	defer assertCalled()
```

Comment:
I believe the comment is out of sync with the code here.  The code sets up a mock server, but the comment suggests its fetching the price.  Maybe I'm missing something?